app/models/user.rb

```diff
@@ -44,7 +9,7 @@ class User < ActiveRecord::Base
          :confirmable,
          :invitable
 
-  validates_presence_of :first_name, :last_name
+  validates_presence_of :first_name, :last_name, :active
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.